src/content/documentation/publish/source-code-submission.md

```diff
@@ -99,6 +97,24 @@ You can add source code to an existing add-on version. To do this, open [My Add-
 
 {% capture content %}
```

Comment:
[Github getting the diff wrong, this would be part of your patch]

I don't really know how things work, but are you sure this is working? There is another section called `content` already defined in another part of this document. Can a page have the different sections with the same name?

Reply:
Here `content` is essentially a variable for the template system, so it can be reused across the different sub-sections of the document. For linking purposes the important part is that we have different IDs assigned to each of these sections (for example, [this line](https://github.com/mozilla/extension-workshop/pull/1758/files/4e18ce5ea3c567f101c651a62f29067a711f5e1f#diff-362cb30ebfed8b5d820e5364d8b8c3120d7278bf64469dc1eb9cc49311c5c157R112)).

Reply:
Thank you.

Reply:
(was put off by the same thing but looked up at other places and then understood hehe 🙃)